### PR TITLE
FE-851: seed data for hash app

### PR DESCRIPTION
**Pull request by alex-e-leon**
## 🌟 What is the purpose of this PR?

The previous seed data sets that I used/were provided to me didn't include enough edge case data for stress testing the ui, so I've gone and built a new seed-data set from scratch that includes more edge cases.

I ended up opting to design the data set around a sales CRM, as a) I think its a potential realistic use-case for hash, with the potential to model various sales/lead generation pipelines down the line to test those features, and b) its an easy category to create fake data for.

I've also opted for generating completely fake data so that there's no issues around data-provenence.

### 🚢 Has this modified a publishable library?

This PR:

- [x] does not modify any publishable blocks or libraries, or modifications do not need publishing

### 📜 Does this require a change to the docs?

The changes in this PR:

- [x] are internal and do not require a docs change

### CODE REVIEW

File: apps/hash-api/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "build:docker": "docker buildx build --tag hash-api --file docker/Dockerfile ../../ --load",
     "codegen": "rimraf './src/**/*.gen.*'; graphql-codegen --config codegen.config.ts",
     "dev": "NODE_ENV=development NODE_OPTIONS=--max-old-space-size=2048 tsx watch --clear-screen=false --import ./src/instrument.mjs ./src/index.ts",
+    "dev:seed-data": "NODE_ENV=development tsx ./src/seed-data/seed-crm-data.ts",
```
Comment (CiaranMn) [inline apps/hash-api/package.json]: There's potential for confusion here with the rest of the content of the `seed-data` folder, which runs whenever the API is started in development. 

I would rename this to `dev:seed-crm-data` to avoid that potential confusion and for alignment with the script being run.

**Review by CiaranMn — COMMENTED** [review]
Very helpful addition! One comment on the command

File: apps/hash-api/package.json (hunk 1 above)
Comment (alex-e-leon) [reply apps/hash-api/package.json]: Good call - done.

**Review by CiaranMn — APPROVED** [review]
